bots/task/github.py

```diff
@@ -60,38 +59,14 @@
 BASE = os.path.normpath(os.path.join(os.path.dirname(__file__), "..", ".."))
 TOKEN = "~/.config/github-token"
 
-# the user name is accepted if it's found in either list
-WHITELIST = os.path.join(BASE, "bots", "whitelist")
-WHITELIST_LOCAL = "~/.config/github-whitelist"
```

Comment:
Do we want to keep this, perhaps (the local override)?

Reply:
We don't use that on the bots, and that's the only relevant place. So I'm in favor of dropping it.

Reply:
`WHITELIST` should be removed together with the whitelist file in the project I think.

 If we'd keep `WHITELIST_LOCAL` I'd avoid doing the override silently.